### PR TITLE
Add console output on Flutter command completion (#527).

### PR DESCRIPTION
**Pull request by pq**
Fixes: #527.

* adds process termination console output
* fixes pubspec finding (when `CommonDataKeys.PSI_FILE` or `LangDataKeys.MODULE` cache-miss)

screen shot 2016-12-12 at 12 25 57 pm

/cc @devoncarew

### CODE REVIEW

File: src/io/flutter/actions/FlutterSdkAction.java [hunk 1]
```diff
@@ -49,15 +50,25 @@
...
+      return findPubspecFrom(ModuleManager.getInstance(project).getModules());
     }
     final VirtualFile file = psiFile.getVirtualFile();
     final VirtualFile contentRoot = ProjectRootManager.getInstance(project).getFileIndex().getContentRootForFile(file);
     return contentRoot == null ? null : contentRoot.findChild(FlutterConstants.PUBSPEC_YAML);
   }
 
+  private static VirtualFile findPubspecFrom(@NotNull Module[] modules) {
+    for (Module module : modules) {
+      final VirtualFile file = findPubspecFrom(module);
```
Comment (devoncarew) [inline src/io/flutter/actions/FlutterSdkAction.java]: This works well if there's one module in a project (or even, one dart module in a project). What should be the behavior if there's more than one matching module? Return the first, or fail (return null)?

**Comment by devoncarew** [comment]
lgtm; one question about how best to support multi-module projects

File: src/io/flutter/actions/FlutterSdkAction.java (hunk 1 above)
Comment (pq) [inline src/io/flutter/actions/FlutterSdkAction.java]: My thinking was to return the first pubspec we find on the assumption that there should not likely be more than one (or that if there were it wouldn't matter).  Not sure if that's the right solution though TBH.

Comment (devoncarew) [inline src/io/flutter/actions/FlutterSdkAction.java]: If there's only one match, returning that makes sense. If there's more than one, we may be better off failing here than running the command for a different module than expected.

Comment (pq) [inline src/io/flutter/actions/FlutterSdkAction.java]: Works for me.  Added a check and failure.

**Comment by pq** [comment]
(Check and error added.)

**Pull request merged**